governor/gov.gno

```diff
@@ -171,11 +179,12 @@ func Vote(id string, yes bool) {
 		panic("Governor not initialized")
 	}
 
-	proposal, exists := gGovernor.proposals[id]
+	proposalValue, exists := gGovernor.proposals.Get(id)
 	if !exists {
 		panic("DOES_NOT_EXIST")
```

Comment:
Please panic with more detail information, in this case such as `id`

Reply:
https://github.com/gnoswap-labs/governance/pull/20/commits/66096f8c7ed485311c7eebf5aa7bfa3720a41c8b